test-project/Assets/EditmodeTests/Subscriptions/RequireablesDisableTests.cs

```diff
@@ -1,117 +1,158 @@
-
-            ConnectionHandler.RemoveComponent(EntityId, Position.ComponentId);
-            ConnectionHandler.RemoveComponent(EntityId, TestCommands.ComponentId);
-            ConnectionHandler.RemoveEntity(EntityId);
-            Update();
-
-            Assert.IsNull(createdGameObject.GetComponent<CommandSender>().Sender);
-            Assert.IsFalse(sender.IsValid);
+            World
+                .Step(world => { world.Connection.CreateEntity(EntityId, GetTemplate()); })
+                .Step(world =>
+                {
+                    var (_, senderBehaviour) = world.CreateGameObject<CommandSender>(EntityId);
+                    return (senderBehaviour: senderBehaviour, sender: senderBehaviour.Sender);
+                })
+                .Step(world =>
+                {
+                    world.Connection.RemoveComponent(EntityId, Position.ComponentId);
+                    world.Connection.RemoveComponent(EntityId, TestCommands.ComponentId);
+                    world.Connection.RemoveEntity(EntityId);
+                })
+                .Step((world, context) =>
+                {
+                    var (senderBehaviour, testCommandsCommandSender) = context;
+                    Assert.IsNull(senderBehaviour.Sender);
+                    Assert.IsFalse(testCommandsCommandSender.IsValid);
+                });
         }
 
         [Test]
         public void CommandReceiver_is_disabled_if_loses_auth()
         {
-            ConnectionHandler.ChangeAuthority(EntityId, TestCommands.ComponentId, Authority.Authoritative);
-            Update();
-
-            createdGameObject = CreateAndLinkGameObjectWithComponent<CommandReceiver>(EntityId);
-            var receiver = createdGameObject.GetComponent<CommandReceiver>().Receiver;
-
-            ConnectionHandler.ChangeAuthority(EntityId, TestCommands.ComponentId, Authority.NotAuthoritative);
-            Update();
-
-            Assert.IsNull(createdGameObject.GetComponent<CommandReceiver>().Receiver);
-            Assert.IsFalse(receiver.IsValid);
+            World
+                .Step(world =>
+                {
+                    world.Connection.CreateEntity(EntityId, GetTemplate());
+                    world.Connection.ChangeAuthority(EntityId, TestCommands.ComponentId, Authority.Authoritative);
+                })
+                .Step(world =>
+                {
+                    var (_, receiverBehaviour) = world.CreateGameObject<CommandReceiver>(EntityId);
+                    return (receiverBehaviour: receiverBehaviour, receiver: receiverBehaviour.Receiver);
+                })
+                .Step(world =>
+                {
+                    world.Connection.ChangeAuthority(EntityId, TestCommands.ComponentId,
+                        Authority.NotAuthoritative);
+                })
+                .Step((world, context) =>
+                {
+                    var (receiverBehaviour, testCommandsCommandReceiver) = context;
```

Comment:
nit: some tests have a new line after the context deconstruction, and some dont
pick one ;)

Reply:
I could only find one w/ a new line and its because it has 4 asserts rather than 2 after it so its harder to read when all jumbled together.

Leaving this as is